Library/Formula/mactelnet.rb

```diff
@@ -0,0 +1,318 @@
+class Mactelnet < Formula
+  homepage "https://github.com/haakonnessjoen/MAC-Telnet"
+  url "https://github.com/haakonnessjoen/MAC-Telnet/archive/v0.4.0.tar.gz"
+  sha1 "57f2a4ae4930e1b4d57c41dd2fafb38ca1142ec5"
+
+  depends_on "gettext"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  # MAC-Telnet 0.4.0 doesn't build against Mac OS X:
+  #   https://github.com/haakonnessjoen/MAC-Telnet/issues/13
+  #
+  # The patch below was submitted as:
+  #   https://github.com/haakonnessjoen/MAC-Telnet/pull/14
+  patch :DATA
```

Comment:
The merge conflicts are due to changes in upstream since the latest release. I think the options are:

1. Point the formula to the current PR patch URL
2. Update the PR to track upstream master and keep the 0.4.0 patch in the formula
3. Update the PR to track upstream master and update the formula to track master

How shall I proceed?